lib/report/account_events.rb

```diff
@@ -0,0 +1,66 @@
+module Report
+  class AccountEvents
+    SESSION_DURATION = 5.hours
```

Comment:
Why five hours out of interest?
Is that just a decent guess at what a window of activity might be?

Reply:
Yeah, it's from an earlier card on thinking about session lengths